src/Pelagos/Bundle/AppBundle/Command/DoiComparisonCommand.php

```diff
@@ -458,8 +458,9 @@ private function doesStringExist(array $metadataElement, string $comparisonEleme
      */
     private function sendEmail(): void
     {
-        $message = \Swift_Message::newInstance()
-            ->setSubject('DOI Sync Log - List of Dois that are out of sync')
+        $message = new \Swift_Message('DOI Sync Log - List of Dois that are out of sync');
+        $message
+            ->setSubject()
```

Comment:
Wouldn't this wipe out the subject set via the Swift_Message constructor?

Reply:
This code has been fixed.